app/javascript/init.js

```diff
@@ -1,16 +1,21 @@
 var de;
 if (! de) { de = {}; }
 if (! de.bender) { de.bender = {}; }
+if (! de.bender.radar ) { de.bender.radar = {}; }
 
+//
 // like on-document-ready but for confluence-pages
 $(document).ready(function () {
     de.bender.prepareSage();
-    new TechRadar(_.first(radar)).create();
+    radar = new TechRadar(_.first(radar));
```

Comment:
Radar is now a global variable if I'm not mistaken - by accident or willfully? If willfully, why?

As you've might guessed I'm a friend of concise code, even call-chains of limited length - as long as they can be understood at a glance. 
I'm of the opinion that the former solution was quite comprehensible already - this solution introduced a global element (which could clash with an already existing radar-variable).